sdk/monitor/Azure.Monitor.Query/src/LogsQueryClient.cs

```diff
@@ -159,7 +159,31 @@ public virtual async Task<Response<LogsQueryResult>> QueryAsync(string workspace
         }
 
         /// <summary>
-        /// Submits the batch query.
+        /// Submits the batch query. Use the <see cref="LogsBatchQuery"/> to compose a batch query.
+        /// <code snippet="Snippet:BatchQuery" language="csharp">
```

Comment:
Should all of these code snippets also be wrapped in `<example>`?

Reply:
I want them to show in VS so embedded the `<code>` directly into the summary.

Reply:
Do both?

Reply:
Then it would be duplicated on the doc page.